src/Database/Model.php

```diff
@@ -1097,20 +1097,6 @@ public function addJsonable($attributes = null)
     // Getters
     //
 
-    /**
-     * Determine if the given attribute will be processed by getAttributeValue().
-     */
-    public function hasAttribute(string $key): bool
-    {
-        return (
-            array_key_exists($key, $this->attributes)
-            || array_key_exists($key, $this->casts)
-            || $this->hasGetMutator($key)
-            || $this->hasAttributeMutator($key)
-            || $this->isClassCastable($key)
-        );
-    }
```

Comment:
Why was this removed?

Reply:
Because this is now in Laravel's Base Model (through the HasAttribute trait)